cg/apps/scout/scout_export.py

```diff
@@ -61,14 +77,14 @@ class ScoutExportCase(BaseModel):
     rank_score_threshold: int = 5
     phenotype_terms: Optional[List[Phenotype]]
     phenotype_groups: Optional[List[Phenotype]]
-    diagnosis_phenotypes: Optional[List[int]]
+    diagnosis_phenotypes: Optional[List[DiagnosisPhenotypes]]
```

Comment:
Could also be an empty array. Would that work in this case?

Reply:
This is more clear and I think it will work fine